src/systemcmds/tests/test_IntrusiveQueue.cpp

```diff
@@ -90,8 +92,20 @@ bool IntrusiveQueueTest::test_push()
 	// pop all elements
 	for (int i = 0; i < 100; i++) {
 		auto node = q1.front();
+
+		if (i < 100 - 1) {
```

Comment:
Here I would always expect it to be not empty.

Reply:
Removed.